### ISSUE TITLE
IPA support

### ISSUE THREAD

**Issue by DavidBitSource** [nvaccess/nvda#11585]
### Is your feature request related to a problem? Please describe.
It is not possible to deal with IPA representations, neither using braille nor speech with NVDA itself.
### Describe the solution you'd like
A complete IPA representation, not based on the braille solutions, since they do lack important features that IPA provides. For text representation I'd suggest a X-SAMPA notations and for speech a description of the symbol and the diacritics.
### Describe alternatives you've considered
There are none so far.

**Comment by zstanecic** [comment]
Hi David,

Espeak supports ipa internally. 

I don’t know how other synthesizers should support this…

**Comment by DavidBitSource** [comment]
IPA should be supported by the screen reader, not just the synthesizer to make it flexible.
I am aware of espeak's capabilities, they are nice, but they do not fit the requirements:
1st it restricts the range of possible use cases.
2nd it is no longer possible to switch between synthesizers.
3rd braille is not considered

**Comment by CyrilleB79** [comment]
Seems a duplicate of #2922.
Please comment there; I will also answer you there.

**Comment by feerrenrut** [comment]
Duplicate of #2922
